### PR TITLE
Add ashleyo.txt - by Ashley O'Mahony

### PR DESCRIPTION
**Pull request by ashomah**

### CODE REVIEW

**Comment by astrojuanlu** [comment]
:tada:

**Comment by astrojuanlu** [comment]
:warning: Next steps :warning: 

Your pull request has been merged to `master`. That means that, if you don't do anything, your local `master` and `ashomah/master` are outdated with respect to `origin/master`. On the other hand, you should avoid working on this branch anymore in the future.

To put yourself in a clean state again, you have to:

1. Click "remove branch" here in this pull request (don't click "remove fork"!)
2. `git checkout master`
3. `git fetch origin` (assuming https://github.com/Juanlu001/ie-mbd-advanced-python/ is `origin` for you)
4. `git merge --ff-only` (this will update your local `master` with `origin/master`, and fail if you accidentally made any commit in `master`)
5. `git fetch -p ashomah` (assuming https://github.com/ashomah/ie-mbd-advanced-python/ is `ashomah` for you) :star2: this will acknowledge the removal of the branch :star2:
6. `git push ashomah master` (this will update your fork with respect to `origin`)

The objective of this process, **which has to be repeated for every pull request**, is to be able to submit a second one without git problems.

~ If in doubt, please ask ~

:rainbow:

File: ashleyo.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a file created by Ashley O'Mahony.
```
Comment (astrojuanlu) [inline ashleyo.txt:1]: Oops, my bad - I didn't notice this file should have been in the `names` directory https://github.com/Juanlu001/ie-mbd-advanced-python/tree/master/names Volunteer to fix it?

Comment (ashomah) [reply ashleyo.txt:1]: I tried, but I see while pulling the master that it's already done 😀

Comment (astrojuanlu) [reply ashleyo.txt:1]: True :slightly_smiling_face:

**Pull request merged**